src/app/modules/shared_models/member_model.nim

```diff
@@ -168,6 +178,18 @@ QtObject:
     self.endInsertRows()
     self.countChanged()
 
+  proc addItems*(self: Model, items: seq[MemberItem]) =
```

Comment:
please return early when `items` is empty. Otherwise we can end up with malformed notifications (e.g. `start = 0` and `end = -1`)